modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -0,0 +1,81 @@
+[[contact-methods-salt]]
+= Salt Bundle
+
+== What is Salt Bundle?
+
+The Salt Bundle is a single binary package containing Salt Minion, Python 3, required Python modules and libraries.
+
+As the Salt Bundle is shipped with Python 3 and all the requirements for Salt. The Bundle is not using the Python installed on the system, so it can be installed on the client which doesn't meet the requirements for Salt version.
```

Comment:
I noticed something weird and just rephrase this as:

```suggestion
As the Salt Bundle is shipped with Python 3 and all the requirements for Salt to run, the Salt Bundle is not using the Python version coming from the system, so it can be installed on clients which doesn't meet the requirements for a given Salt version.
```